config/secrets.yml

```diff
@@ -32,16 +32,19 @@ default: &default
       client_id: <%= ENV["OMNIAUTH_GOOGLE_CLIENT_ID"] %>
       client_secret: <%= ENV["OMNIAUTH_GOOGLE_CLIENT_SECRET"] %>
   geocoder:
-    here_app_id: <%= ENV["HERE_APP_ID"] %>
-    here_app_code: <%= ENV["HERE_APP_CODE"] %>
+    here_api_key: <%= ENV["HERE_API_KEY"] %>
```

Comment:
Can you check if this key is available in the server? Otherwise we need to add it before merging this PR